src/pages/TokenDetails/index.tsx

```diff
@@ -79,10 +80,29 @@ export default function TokenDetails() {
   const location = useLocation()
   const { tokenAddress } = useParams<{ tokenAddress?: string }>()
   const token = useToken(tokenAddress)
+  const tokenWarning = tokenAddress ? checkWarning(tokenAddress) : null
+  const isBlockedToken = !!tokenWarning && !tokenWarning?.canProceed
```

Comment:
Would this be more clearly expressed as

    const isBlockedToken = tokenWarning?.canProceed === false